library/spdm_requester_lib/libspdm_req_send_receive.c

```diff
@@ -588,8 +588,12 @@ libspdm_return_t libspdm_send_spdm_request(libspdm_context_t *spdm_context,
             spdm_context->last_spdm_request_size = request_size;
         }
 
-        status = libspdm_handle_large_request(
-            spdm_context, session_id, request_size, request);
+        if (request_size > spdm_context->connection_info.capability.max_spdm_msg_size) {
+            return LIBSPDM_STATUS_SEND_FAIL;
```

Comment:
@steven-bellock , do you think if we need a special error code?

Reply:
Yes, `LIBSPDM_STATUS_SEND_FAIL` is for transport issues, but this is a buffer-too-small issue. So let's make 
>/* Message is too large to send to peer. */
#define LIBSPDM_STATUS_PEER_BUFFER_TOO_SMALL \

Reply:
Thanks. I have added the `LIBSPDM_STATUS_PEER_BUFFER_TOO_SMALL` MACRO.